src/transform.ts

```diff
@@ -110,20 +111,17 @@ export interface WindowFieldDef {
   op: AggregateOp | WindowOnlyOp;
 
   /**
-   *  Parameter values for the window functions. Parameter value can be null for operations that do not accept a
-   *  parameter.
+   *  Parameter values for the window functions. Parameter value can be omitted for operations that do not accept a parameter.
    */
   param?: number;
 
   /**
-   * The data fields for which to compute aggregate or window functions. Field can be omitted for operations that do not
-   * operate over a specific data field, including count, rank, and dense_rank.
+   * The data field for which to compute the aggregate or window function. This can be null for functions that do not operate over a field such as `count`, `rank`, `dense_rank`.
    */
   field?: string;
 
   /**
-   * The output name for each field. If none is defined will use the format op_field. For example, count_field for count,
-   *  and sum_field for sum.
+   *  The output name for each field. If non specified will use the format `window_op_field` for example, `count_field` for count and `sum_field` for sum.
```

Comment:
Can you make as required by removing question mark below?   (Yes  -- we should  still autogenerate it if `as` is missing, but JSON schema to enforce people to specify it, so that people write better specs.)